tests/mocks/sublime_mock.py

```diff
@@ -29,3 +29,11 @@ def load_settings(*args):
         'included_files': [],
         'excluded_files': []
     })
+
+
+def encode_value(*args):
```

Comment:
replace this function by:

```python
def encode_value(data):
    return json.dumps(data)
```

With this, you will be able to replicate the behavior of `sublime.encode_value`